recipes/default.rb

```diff
@@ -129,7 +129,7 @@
   options ['copytruncate', 'missingok', 'compress', 'notifempty', 'delaycompress']
   frequency 'daily'
   rotate node['opsline-chef-client']['logrotate']['days']
-  enable node['opsline-chef-client']['logrotate']['enabled'] ? :create : :delete
+  enable node['opsline-chef-client']['logrotate'].has_key?('enabled') ? node['opsline-chef-client']['logrotate']['enabled'] : false
```

Comment:
yes, but if it's somehow set to `nil`, the validation will fail... an additional safeguard